docs/core/testing/unit-testing-mstest-configure.md

```diff
@@ -100,3 +100,99 @@ Each element of the file is optional because it has a default value.
+
+#### Run settings
+
+| Entry | Default | Description |
+|-------|---------|-------------|
+| considerEmptyDataSourceAsInconclusive | false | When set to `true`, an empty data source is considered as inconclusive. |
+| considerFixturesAsSpecialTests | false | To display `AssemblyInitialize`, `AssemblyCleanup`, `ClassInitialize`, `ClassCleanup` as individual entries in Visual Studio and Visual Studio Code `Test Explorer` and _.trx_ log, set this value to **true**. |
+| mapInconclusiveToFailed | false | If a test completes with an inconclusive status, it's mapped to the skipped status in **Test Explorer**. If you want inconclusive tests to be shown as failed, set the value to **true**. |
+| mapNotRunnableToFailed | true | A value indicating whether a not runnable result is mapped to failed test. |
+| treatClassAndAssemblyCleanupWarningsAsErrors | false | To see your failures in class cleanups as errors, set this value to **true**. |
+| treatDiscoveryWarningsAsErrors | false | To report test discovery warnings as errors, set this value to **true**. |
+
+#### Timeout settings
+
+| Entry | Default | Description |
+|-------|---------|-------------|
+| assemblyCleanup | 0 | Specify globally the timeout to apply on each instance of assembly cleanup method. |
+| assemblyInitialize | 0 | Specify globally the timeout to apply on each instance of assembly initialize method. |
+| classCleanup | 0 | Specify globally the timeout to apply on each instance of class cleanup method. |
+| classInitialize | 0 | Specify globally the timeout to apply on each instance of class initialize method. |
+| test | 0 | Specify globally the test timeout. |
+| testCleanup | 0 | Specify globally the timeout to apply on each instance of test cleanup method. |
+| testInitialize | 0 | Specify globally the timeout to apply on each instance of test initialize method. |
+| useCooperativeCancellation | false | When set to `true`, in case of timeout, MSTest will only trigger cancellation of the `CancellationToken` but will not stop observing the method. This is more performant but relies on user to correctly flow the token through all paths. |
+
+> [!NOTE]
+> `[Timeout]` attribute specified on a method overrides the global timeout. For example, `[Timeout(1000)]` on a method marked with [AssemblyCleanup] will override the global `assemblyCleanup` timeout.
+
+### Example *testconfig.json* file
+
+The following json shows the contents of a typical *.testconfig.json* file. Copy this code and edit it to suit your needs.
+
+Each element of the file is optional because it has a default value.
+
+```json
+{
+  "mstest": {
+    "execution": {
+        "mapInconclusiveToFailed" : true,
+        "disableAppDomain": true,
+        "considerFixturesAsSpecialTests" : false,
+    },
+    "parallelism" : {
+        "enabled": true,
+        "scope": "method",
+    },
+    "output": {
+        "captureTrace": false
+    }
+  }
+}
+```
```

Comment:
For clarify, I'd add "platformOptions" to the example, and add comments in JSON to clarify that "platformOptions" are applicable to any testing framework that uses the new platform, while the "mstest" part is specific to MSTest.